openedx/core/djangoapps/oauth_dispatch/tests/mixins.py

```diff
@@ -23,47 +24,23 @@ def assert_valid_jwt_access_token(self, access_token, user, scopes=None, should_
         Returns:
             dict: Decoded JWT payload
         """
-        scopes = scopes or []
-        audience = aud or settings.JWT_AUTH['JWT_AUDIENCE']
-        secret_key = secret or settings.JWT_AUTH['JWT_SECRET_KEY']
-        issuer = settings.JWT_AUTH['JWT_ISSUER']
```

Comment:
Author's note: I didn't spot any documentation on this, but the code in https://github.com/openedx/edx-drf-extensions/blob/a8bf162aa41f6b1b629828c41adaf9b647971f57/edx_rest_framework_extensions/settings.py calls these settings deprecated and prefers that people use a `JWT_ISSUERS` list.